frontend/src/Components/Controls/SocialList.tsx

```diff
@@ -5,15 +5,24 @@ export type SocialListing = {
     text: string
 }
 
-export function SocialList({ listings, heading }: { listings: Array<SocialListing>, heading: string }) {
+type SocialListProps = {
+    listings: Array<SocialListing>,
+    heading: string
+}
+
+export const SocialList: React.FC<SocialListProps> = ({ listings, heading }) => {
+    // former code. to be removed later
+    // export function SocialList({ listings, heading }: { listings: Array<SocialListing>, heading: string }) {
+
+
     return (
         <div style={SocialListStyle}>
             <h2 style={SocialHeadingStyle}>
                 {heading}
             </h2>
             {
-                listings.map( (listing, index) =>
-                    <a key={index} target="_blank" href={listing.hrefUrl} style={SocialListAttributeStyle}>
+                listings.map((listing, index) =>
+                    <a key={index} target="_blank" href={listing.hrefUrl} style={SocialListAttributeStyle} rel="noreferrer">
```

Comment:
What is the intension behind adding `rel="noreferrer"` to the hyperlinks here? That would cause the site being navigated to to not know what site was navigated from. It isn't a major implication, but I could potentially see future cases where we would want websites to know that their visitors came from tafers.net.

Reply:
Using target="_blank" without rel="noreferrer" (which implies rel="noopener") is a security risk in older browsers: see https://mathiasbynens.github.io/rel-noopener/#recommendationseslintreact/jsx-no-target-blank
It's your call